packages/app/src/feature-flags/constants/feature.enum.ts

```diff
@@ -16,4 +16,5 @@ export enum Feature {
   ProductionReleaseOnly = "production-release-only",
   TestProductionReleaseOnly = "test-production-release-only",
   AllReleaseListAvailable = "all-release-list-available",
+  ReadAndUnreadMessages = "mark-and-unread-messages",
```

Comment:
Did you mean _read_ ?
```suggestion
  ReadAndUnreadMessages = "read-and-unread-messages",
```